### DISCUSSION TITLE
Is there a way to assign a initial window position for a App?

### DISCUSSION THREAD

**Discussion by BenasdTW** [emilk/egui#1182] [Q&A]
Title

**Comment by emilk** [comment]
No, but it would be an easy thing to add to https://docs.rs/epi/latest/epi/struct.NativeOptions.html

Just follow what https://github.com/emilk/egui/pull/1171 did!

**Reply by BenasdTW** [reply]
I see, that will be awesome if you can add it! Excited to see it in the next version of Egui.
